client/landing/stepper/declarative-flow/internals/steps-repository/domains/domain-form-control.tsx

```diff
@@ -260,12 +272,17 @@ export function DomainFormControl( {
 	};
 
 	let content;
+	let sideContent;
 	if ( showUseYourDomain ) {
 		content = renderYourDomainForm();
 	} else {
 		content = renderDomainForm();
 	}
 
+	if ( isReskinnedSupportedFlow() && ! showUseYourDomain ) {
```

Comment:
Only show the right side content on the domain pages if the flow is supported and the user has not clicked on the "Use a domain I own" link

Reply:
On line 244, we pass the equivalent of `sideContent` (or `getSideContent()` ) to `renderDomainForm`. Seems like it would be possible here to render sideContent via line 244 and via line 290 at the same time? Can you provide a bit of clarity on the interplay between renderDomainForm, renderDomainForm, and sideContent?

Reply:
Just to follow up on this, I discussed with @agrullon95. It looks like the RegisterDomainStep *does* have the capacity to show the sideContent, and we could plausibly have leveraged that existing capacity. That domain actually shows the side content, but only for mobile. So to get it to show, we'd need to update the RegisterDomainStep directly. I think what we're thinking here is to avoid adjusting that component for the sake of backwards compatibility. The approach taken here, where our edits are confined to the stepper page, makes it less likely we'd introduce unanticipated effects for the RegisterDomainStep component.